<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PdfExportDialog</class>
 <widget class="QDialog" name="PdfExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>495</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>522</width>
    <height>495</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>522</width>
    <height>495</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FalconBoard - PDF Export</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbPaper">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Screen</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Diagonal</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="cbScreenResolution">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="placeholderText">
         <string>Select Screen Resolution</string>
        </property>
        <item>
         <property name="text">
          <string>2840 x 2160 (16:9 - 4K)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3440 x 1440 (21:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2560 x 1440 (16:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2560 x 1080 (21:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2048 x 1152 (16:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1920 x 1200 (16:10)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1920 x 1080 (16:9 - Full HD)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1680 x 1050 (16:10)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1600 x 900 (16:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1536 x 864 (16 x 9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1440 x 900 (16:10)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1366:768 (16:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1360 x 768 (16:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1280 x 1024 (5:4)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1280 x 800 (16:10)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1280 x 720 (16:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1024 x 768 (4:3)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>800 x 600 (4:3)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>640 x 360 (16:9</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="edtScreenDiag">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>24</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>107</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="cbUnit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>inch</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mm</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="sbHorizPixels">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="suffix">
         <string> pixels</string>
        </property>
        <property name="minimum">
         <number>300</number>
        </property>
        <property name="maximum">
         <number>16355</number>
        </property>
        <property name="singleStep">
         <number>11</number>
        </property>
        <property name="value">
         <number>1920</number>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>or Horizontal Pixel Count to print on one page</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkWhiteBackground">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>&amp;White background</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QCheckBox" name="chkGrayscale">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>&amp;Grayscale</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>262</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkPrintBackgroundImage">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Print &amp;Background Image</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="chkGrid">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>G&amp;rid</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>290</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkDontPrintImages">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Do not print &amp;images</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>93</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbExportPdf">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>PDF</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Paper size:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbPdfPaperSize">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>A7 ( 74 x 105 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A6 (105 x 148 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A5 (148 x 210 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A4 (210 x 297 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A3 (297 x 420 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A2 (420 x 594 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A1 (594 x 841 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A0 (841 x 1189 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B7 (88 x 125 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B6 (125 x 176 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B5 (176 x 250 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B4 (250 x 353 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B3 (353 x 500 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B2 (500 x 707 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B1 (707 x 1000 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B0 (1000 x 1414 mm)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>US Letter (8 1/2 x 11 in)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Legal (2 1/2 x 14 in)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Stationary (5 1/2 x 8 1/2 in)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2" rowspan="3">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Resolution</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="rb300">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>300 dpi</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb600">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>600 dpi</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb1200">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>1200 dpi</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Paper Orientation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbOrientation">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Portrait</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Landscape</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>82</height>
         </size>
        </property>
        <property name="title">
         <string>Margins</string>
        </property>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>103</width>
           <height>24</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Margin Left and Right</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="sbMarginLR">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>49</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::DefaultStepType</enum>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>111</width>
           <height>24</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Margin Top and Bottom</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="sbMarginTB">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>50</y>
           <width>49</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
        <widget class="QComboBox" name="cbPdfUnit">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>30</y>
           <width>49</width>
           <height>24</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>cm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>inch</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExportPdf">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Export PDF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>PdfExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
